src/m365/planner/commands/task/task-get.ts

```diff
@@ -15,6 +15,7 @@ interface CommandArgs {
 }
 
 interface Options extends GlobalOptions {
+  taskId?: string; // This option has been added to support task details get alias. Needs to be removed when deprecation is removed. 
```

Comment:
Let's add this option to the command's docs including a remark that it's deprecated and that it will be removed in the next major version.